<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateWindow</class>
 <widget class="QMainWindow" name="UpdateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>423</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>758</width>
    <height>423</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>758</width>
    <height>423</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Update Virus Signatures</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resourses.qrc">
    <normaloff>:/clamguard/Gerald-G-Clam-Security-Guard-ico32x32.png</normaloff>:/clamguard/Gerald-G-Clam-Security-Guard-ico32x32.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblUpdate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Virus Signatures</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtUpdate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>731</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>380</y>
      <width>87</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resourses.qrc"/>
 </resources>
 <connections/>
</ui>
